### PR TITLE
Cloudbot UX Fixes

### PR DESCRIPTION
**Pull request by AnkhHeart**
- UX Fixes in relation to colors in the Cloudbot Section (Cards, Tables, Text, ...)
- Improved Variable list
- Improved Default Commands list
- Using Empty Sections from beaker instead of custom images to give intructions to users
- Fixed window height in the Poll & betting profiles

### CODE REVIEW

File: app/components/page-components/Chatbot/ChatbotCommandVariables.vue.ts [hunk 1]
```diff
@@ -1,10 +1,50 @@
...
+    }
+
+    const filteredVariables = mapValues(grouped, (section, slug) => {
+      return pickBy(
+        mapValues(section, (variable: any) => {
+          const found =
+            slug.toLowerCase().indexOf(this.searchQuery.toLowerCase()) > -1 ||
+            variable.variable.toLowerCase().indexOf(this.searchQuery.toLowerCase()) > -1 ||
+            variable.example.toLowerCase().indexOf(this.searchQuery.toLowerCase()) > -1 ||
+            variable.result.toLowerCase().indexOf(this.searchQuery.toLowerCase()) > -1;
```
Comment (holiber) [inline app/components/page-components/Chatbot/ChatbotCommandVariables.vue.ts]:
```
const found = [slug, variable.variable, variable.example, variable.result]
 .map(str => str.toLowerCase())
 .find(str => str.includes(this.searchQuery.toLowerCase()))
```

File: app/components/page-components/Chatbot/ChatbotCommandVariables.vue.ts [hunk 2]
```diff
@@ -1,10 +1,50 @@
...
+          grouped[tag].push(variable);
+        } else {
+          grouped[tag].push(variable);
+        }
+      }
+    }
+
+    const filteredVariables = mapValues(grouped, (section, slug) => {
+      return pickBy(
+        mapValues(section, (variable: any) => {
```
Comment (holiber) [inline app/components/page-components/Chatbot/ChatbotCommandVariables.vue.ts]: can we get rid of `any` here?

File: app/services/chatbot/chatbot-common.ts [hunk 3]
```diff
@@ -330,7 +330,7 @@ export class ChatbotCommonService extends PersistentStatefulService<IChatbotComm
   openGamblePreferencesWindow() {
     this.windowsService.showWindow({
       componentName: 'ChatbotGamblePreferencesWindow',
-      title: $t('Chatbot Gamble Preferences'),
+      title: $t('Cloudbot Gamble Preferences'),
```
Comment (holiber) [inline app/services/chatbot/chatbot-common.ts]: We are in the process of translation now. Please update translation strings in `en-US` folder

File: app/components/page-components/Chatbot/ChatbotCommandVariables.vue.ts (hunk 1 above)
Comment (AnkhHeart) [reply app/components/page-components/Chatbot/ChatbotCommandVariables.vue.ts]: Done, it's in the latest commit. Ty for letting me know of a cleaner way to do this :)

File: app/services/chatbot/chatbot-common.ts (hunk 3 above)
Comment (AnkhHeart) [reply app/services/chatbot/chatbot-common.ts]: Done.

File: app/components/page-components/Chatbot/ChatbotCommandVariables.vue.ts (hunk 2 above)
Comment (AnkhHeart) [reply app/components/page-components/Chatbot/ChatbotCommandVariables.vue.ts]: Done.

File: app/i18n/en-US/chatbot.json [hunk 4]
```diff
@@ -314,26 +314,68 @@
...
+  "Reset": "Reset",
+  "Minimum Amount of Emotes": "Minimum Amount of Emotes",
+  "Maximum Amount of Emotes": "Maximum Amount of Emotes",
+  "1-25": "1-25",
+  "26-50": "26-50",
+  "51-75": "51-75",
+  "76-98": "76-98",
+  "99-100": "99-100",
+  "Tie Message": "Tie Message",
+  "#": "#"
```
Comment (holiber) [inline app/i18n/en-US/chatbot.json]: Numbers and special characters must not be a part of translation

Comment (holiber) [reply app/i18n/en-US/chatbot.json]: Just use these strings without `$t` wrapper

Comment (AnkhHeart) [reply app/i18n/en-US/chatbot.json]: Done

**Pull request merged**